gesund/validation.py

```diff
@@ -91,7 +91,7 @@ def run_metrics(args):
 
     return metrics
 
-def plotting_metrics(metrics, filtering_meta=None):
+def plotting_metrics(metrics, args, filtering_meta=None):
```

Comment:
i suggest, we could pass explicitly `args["plot_configs"]`, so that there is no need for `args.get`

Reply:
I tested like replaced args.get('plot_configs', {}) with args['plot_configs'] in the function, it gives the same results.